apps/condo/domains/news/access/NewsItemScope.js

```diff
@@ -41,17 +41,14 @@ async function canManageNewsItemScopes ({ authentication: { item: user }, origin
     if (user.isAdmin) return true
 
     if (user.type === STAFF) {
-        let newsItemId = get(originalInput, ['newsItem', 'connect', 'id'])
         if (operation === 'update') {
-            if (!itemId) return false
-
-            if (!newsItemId) {
-                const newsItemScope = await getById('NewsItemScope', itemId)
-                newsItemId = newsItemScope.newsItem
-            }
+            return false
```

Comment:
you can change the unitName if you want? (optional)

Reply:
If receivers are changed, we create a new set of scopes.